assets/js/modules/adsense/dashboard/adsense-module-status.js

```diff
@@ -55,7 +55,7 @@ class AdSenseModuleStatus extends Component {
 
 	async componentDidMount() {
 		let existingTag = await getExistingTag( 'adsense' );
-		existingTag = existingTag.length ? existingTag : false;
+		existingTag = existingTag ? existingTag : false;
```

Comment:
I think this line can simply be removed. The return value of `getExistingTag()` should only be a non-zero length string, or `null`. All references beyond this point check for a truthy value so there is nothing special about it being `=== false`.

We could potentially update the default value of `existingTag` to be `null` as well in `getAdSenseAccountStatus` but it shouldn't make a difference.

@adamsilverstein if you can double check this, that would be great ☝️ 

```suggestion
```

Reply:
Aha, nice!